tests/logs/delete/test_delete_packages/test_delete_rpms_with_all_repos.txt

```diff
@@ -0,0 +1,37 @@
+[    INFO] Delete files: started
+[    INFO] Get files: started
+[ WARNING] Requested unit(s) don't exist as file: bash-1.23-1.test8_x86_64.rpm, dash-2.25-1.test8_x86_64.rpm
+[    INFO] 0 unit(s) found for deletion
+[    INFO] Get files: finished
+[ WARNING] No units to remove from *, all-rpm-content-xp
+[    INFO] Unassociate files: started
+[ WARNING] Nothing mapped for removal
+[    INFO] Unassociate files: finished
+[    INFO] Record push items: started
+[    INFO] Record push items: finished
+[    INFO] Delete files: finished
+[    INFO] Delete RPMs: started
+[    INFO] Get RPMs: started
+[    INFO] 2 unit(s) found for deletion
+[    INFO] Get RPMs: finished
+[ WARNING] No units to remove from *
```

Comment:
The warning seems undesirable.  I think it's going to be produced every time `*` is used even if content is actually being removed?

This problem might also be avoided if the code worked more like the psuedo-code from the earlier thread.